dokka-subprojects/plugin-base-frontend/src/main/ui-kit/global.scss

```
@@ -7,13 +7,23 @@
   --color-background: var(--color-background-page);
   --focus-outline: 4px solid var(--color-key-blue-50);
   --content-max-width: 1128px;
+  --text-color: var(--color-text);
+  --text-outine: var(--color-b20);
```

Is it `text-outline`? or even `text-underline`?
```suggestion
  --text-outline: var(--color-b20);
```